README.md

```diff
@@ -122,7 +122,13 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Contribution
 
-Any contributions to Optuna are welcome! When you send a pull request, please follow the
+Any contributions to Optuna are welcome!
+
+If you are a new contributor, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
```

Comment:
A nit.
```suggestion
If you are a new contributor, please check the [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
```

Reply:
Ops, did this suggestion get lost? Maybe the order in which my suggestions were applied.

Reply:
I'm sorry but I overlooked this comment since the commit suggestion button is disabled in the GitHub UI. I applied it manually in https://github.com/optuna/optuna/pull/1597/commits/dca1241f1468666c9072a1ca212e16e4e61ac77a.